src/bsoncxx/exception/libbson_error.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 MongoDB Inc.
```

Comment:
Is this header public or private? If it is public, it leaks that we are using libbson internally.